geant-incubator/Issues/surf-sram/description.xml

```diff
@@ -0,0 +1,186 @@
+	</FAQIntro>
+	<FAQPurpose>
+		<en>By adding credentials from from your research collaboration(s) to your IRMA Wallet, you may gain access to services provided by your research collaboration.</en>
+	</FAQPurpose>
+	<FAQContent>
+		<en>You may add the following data:\n•name(s)\n• e-mail addres(s)\n• affiliation\n• identifier(s)\n• your roles and rights in the context of your collaboration\n• assurance information</en>
+	</FAQContent>
+	<FAQHowto>
+		<en>If you are a member of a research collaboration that has is using SURF SRAM to manage access to it services, you may add personal data from this collaboration to your IRMA app.\n\n Complete the following steps.\n1. Select your own research collaboration\n2. Log in at SRAM in the usual manner.\n3. Confirm that you wish to add your data to IRMA. \n\nYou now receive a card with your Research community attributes.</en>
+	</FAQHowto>
+    <Attributes>
+	   <!-- user identifier(s)  -->
+       <Attribute id="subject-id" optional="true">
+          <Name>
+             <en>subject Identifier</en>
+          </Name>
+          <Description>
+             <en>A unique identification of subjects. May be either a globally-unique identifier, or a pairwise identifier suitable for more specialized uses.</en>
+          </Description>
+       </Attribute>
+       <Attribute id="eduPersonPrincipalName" optional="true">
+          <Name>
+             <en>Principal Name</en>
+          </Name>
+          <Description>
+             <en>A scoped identifier for a person.</en>
+          </Description>
+       </Attribute>
+       <Attribute id="eduPersonOrcid" optional="true">
+          <Name>
+             <en>Orcid</en>
+          </Name>
+          <Description>
+             <en>ORCID iDs are persistent digital identifiers for individual researchers. Their primary purpose is to unambiguously and definitively link them with their scholarly work products. ORCID iDs are assigned, managed and maintained by the ORCID organization.</en>
+          </Description>
+       </Attribute>        
+       <Attribute id="eduperson_unique_id" optional="true">
+          <Name>
+             <en>Unique ID</en>
+          </Name>
+          <Description>
+             <en>Unique persistent identifier for the user</en>
+          </Description>
+       </Attribute>
+       <Attribute id="UID" optional="true">
+          <Name>
+             <en>UID</en>
+          </Name>
+          <Description>
+             <en>Short, human-readably username for login to a backend systems</en>
+          </Description>
+       </Attribute>
+       <Attribute id="voperson_external_id" optional="true">
+          <Name>
+             <en>External ID</en>
+          </Name>
+          <Description>
+             <en>Identifier from the user's home organization</en>
+          </Description>
+       </Attribute>
+       <Attribute id="ssh_public_key" optional="true">
+          <Name>
+             <en>Ssh public key</en>
```

Comment:
Are you planning to put an entire SSH public key in this attribute? The IRMA app will then show the entire thing, that'll probably look very ugly. (Also, nitpick: `Ssh` is usally capitalized, `SSH`.)

Reply:
This is currently I think a generic challenge with IRMA: sometimes information that needs to be provided to the verifier is a 'techncial' identifier, like in this case the SSH public key. We have some other identifiers with similar challenges. These strings are indeed not really intended to be shown to users, however, you do want to express to the user the fact that this identifier is requested. 
In our existing AAI infrastructures we typically show a user friendly display name fur such identifiers and only show the real string value of the user wants to see the technical details.

SSH indeed needs to be capitalized